test/common/upstream/load_balancer_benchmark.cc

```diff
@@ -361,6 +478,33 @@ BENCHMARK(BM_MaglevLoadBalancerWeighted)
     ->Args({500, 95, 75, 25, 10000})
     ->Unit(benchmark::kMillisecond);
 
+void BM_RoundRobinLoadBalancerWeighted(benchmark::State& state) {
+  for (auto _ : state) {
+    state.PauseTiming();
+    const uint64_t num_hosts = state.range(0);
+    const uint64_t weighted_subset_percent = state.range(1);
+    const uint64_t weight = state.range(2);
+
+    RoundRobinTester tester(num_hosts, weighted_subset_percent, weight);
+
+    // We are only interested in timing the initial build.
+    state.ResumeTiming();
+    tester.initialize();
+    state.PauseTiming();
+
+    // Exclude destructors from timing.
+  }
+}
+BENCHMARK(BM_RoundRobinLoadBalancerWeighted)
```

Comment:
Is there no ChooseHost for weighted?

Reply:
The ChooseHost benchmark covers both unweighted cases like:
  Args({500, 0, 1})
and weigted cases like:
  Args({500, 50, 75})

Perhaps the weighted and non-weigted build benchmarks should be merged?

Reply:
Oh, I see what you mean.  BM_MaglevLoadBalancerWeighted does more than just build, so in order to be consistent with those I should also do some chooseHost in here.

Reply:
Thoughts on trying to mimic what the other Weighted benchmarks do vs excluding this benchmark for RoundRobin?

Reply:
I think what you have is fine actually, maybe just document with some comments this fact in `BM_RoundRobinChooseHost`